src/module-elasticsuite-virtual-category/Model/Rule.php

```diff
@@ -189,6 +198,7 @@ public function getCategorySearchQuery($category, $excludedCategories = []): ?Qu
             ]
         );
```

Comment:
Shouldn't the value of the setting be taken into account for the cacheKey ?

Reply:
But if we add it to the cache key, it will invalidate all the category cache if the user update this config as none of the cache key will match anymore ?